doc/packages.md

```diff
@@ -0,0 +1,314 @@
+# gRPC ASP.NET Core packages
+
+## Overview
+
+![image](images/packages.png)
+
+## Design considerations
+
+### Grpc.AspNetCore
+
+We envision shipping both a managed client and managed server. Since gRPC services often act as both client and server, we want to make the consumption of both components as frictionless as possible. In addition to the server and client packages, the metapackage will also take dependencies on tooling support (code gen from Grpc.Tools and ServiceReference support from Microsoft.Extensions.ApiDescription.Design) and the default message protocol (Google.Protobuf).
+
+We want to distinguish these from the native clients and server so the package names will be prefixed with Grpc.AspNetCore/Grpc.NetCore. We don't expect most users to take a direct dependency on these packages so we will use a three-part name.
+
+We are committed to delivering the managed server experience Microsoft.AspNetCore.Server functionalities in ASP.NET Core 3.0 timeframe. We will strive to also deliver the mananged client experience in 3.0.
```

Comment:
Perhaps the plans and timeline should be described in a separate .md file
(fine to move there later).

Reply:
I'm not sure there would be enough content for a complete timeline doc so I just included this one line here. cc @shirhatti if you disagree.

Reply:
@shirhatti

Reply:
@shirhatti